spec/modals/category_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.feature Category, type: :model do
+  background do
+    @user = User.create!(name: 'Jake', email: 'jake@mail.com', password: 'jake123')
+
+    @category = @user.categories.create!(name: 'Musica')
+
+    @category.icon.attach(
+      io: File.open(Rails.root.join('spec', 'pics', 'libro.jpeg')),
+      filename: 'libro.jpeg',
+      content_type: 'application/jpeg'
+    )
+    @category.save!
+  end
+
+  it 'should be valid' do
+    expect(@category).to be_valid
+  end
```

Comment:
- [ ] One of your tests is failing. Kindly resolve it.

  ![tchilo-test](https://user-images.githubusercontent.com/36542986/164298403-6ea6ef60-bce8-4fee-b804-0d4f5f0a1436.png)